website/src/components/lifeCycle/styles.module.css

```diff
@@ -8,7 +8,6 @@
     margin-left: 12px; /* Margin to separate from the header text */
     vertical-align: middle; /* Align with the title */
     display: inline-block; /* Use inline-block for better control */
-    text-transform: capitalize; /* Uppercase text */
```

Comment:
Removed this because when I switched to two words, it was capitalizing both words.